website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

```diff
@@ -66,91 +68,100 @@ clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips).
 
 Given you have the [official Helm chart](/docs/k8s/helm) installed with the default values, do the following to join an external client agent.
 
-  1. Make sure the pod IPs of the clients and servers in Kubernetes are
-  routable from the VM and that the VM can access port 8301 (for gossip) and
-  port 8300 (for server RPC) on those pod IPs.
-
-  1. Make sure that the client and server pods running in Kubernetes can route
-  to the VM's advertise IP on its gossip port (default 8301).
-
-  2. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config` on the external VM.
-
-  2.  On the external VM, run:
-  ```bash
-  consul agent \
-    -advertise="$ADVERTISE_IP" \
-    -retry-join='provider=k8s label_selector="app=consul,component=server"'
-    -bind=0.0.0.0 \
-    -hcl='leave_on_terminate = true' \
-    -hcl='ports { grpc = 8502 }' \
-    -config-dir=$CONFIG_DIR \
-    -datacenter=$DATACENTER \
-    -data-dir=$DATA_DIR \
-  ```
-
-  3. Check if the join was successful by running `consul members`. Sample output:
-  ```shell-session
-  / $ consul members
-  Node                                           Address           Status  Type    Build  Protocol  DC   Segment
-  consul-consul-server-0                         10.138.0.43:9301  alive    server  1.9.1  2         dc1  <all>
-  external-agent                                 10.138.0.38:8301  alive    client  1.9.0  2         dc1  <default>
-  gke-external-agent-default-pool-32d15192-grs4  10.138.0.43:8301  alive    client  1.9.1  2         dc1  <default>
-  gke-external-agent-default-pool-32d15192-otge  10.138.0.44:8301  alive    client  1.9.1  2         dc1  <default>
-  gke-external-agent-default-pool-32d15192-vo7k  10.138.0.42:8301  alive    client  1.9.1  2         dc1  <default>
-  ```
+1. Make sure the pod IPs of the clients and servers in Kubernetes are
+   routable from the VM and that the VM can access port 8301 (for gossip) and
+   port 8300 (for server RPC) on those pod IPs.
+
+1. Make sure that the client and server pods running in Kubernetes can route
+   to the VM's advertise IP on its gossip port (default 8301).
+
+1. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config` on the external VM.
+
+1. On the external VM, run:
+
+```bash
+consul agent \
+  -advertise="$ADVERTISE_IP" \
+  -retry-join='provider=k8s label_selector="app=consul,component=server"'
+  -bind=0.0.0.0 \
+  -hcl='leave_on_terminate = true' \
+  -hcl='ports { grpc = 8502 }' \
+  -config-dir=$CONFIG_DIR \
+  -datacenter=$DATACENTER \
+  -data-dir=$DATA_DIR \
+```
```

Comment:
Nesting this under the numbered items - prettier doesn't always get this right when the indentation base isn't clear.

````suggestion
    ```bash
    consul agent \
      -advertise="$ADVERTISE_IP" \
      -retry-join='provider=k8s label_selector="app=consul,component=server"'
      -bind=0.0.0.0 \
      -hcl='leave_on_terminate = true' \
      -hcl='ports { grpc = 8502 }' \
      -config-dir=$CONFIG_DIR \
      -datacenter=$DATACENTER \
      -data-dir=$DATA_DIR \
    ```
````